osmaxx/excerptexport/models/export.py

```diff
@@ -163,6 +163,10 @@ def is_status_final(self):
         return self.status in self.FINAL_STATUSES
 
     @property
+    def can_be_deleted(self):
+        return self.is_status_final or self.update_overdue
```

Comment:
:+1: for having logic in model property or method.

Alternative name suggestion: `is_deletable`. (Current name is fine, too, though.)